SwiftUI_Homework/SwiftUI_Homework/TabView/PrivacyView.swift

```diff
@@ -0,0 +1,193 @@
+//
+//  Privacy.swift
+//  SwiftUI_Homework
+//
+//  Created by Анастасия Козлова on 22.01.2023.
+//
+
+import SwiftUI
+
+/// Экран выбора тарифа
+struct PrivacyView: View {
+    
+    // MARK: - Private Constants
+    
+    private enum Constants {
+        static let blueColor = Color(#colorLiteral(red: 0.2569369539, green: 0.8115655123, blue: 0.9686274529, alpha: 0.4798957875))
```

Comment:
```suggestion
        static let blueColor = Color(#colorLiteral(red: 0.2569369539, green: 0.8115655123, blue: 0.9686274529, alpha: 0.4798957875))
```
то же самое, лучше в ассеты